src/main/java/uk/gov/companieshouse/confirmationstatementapi/interceptor/LoggingInterceptor.java

```diff
@@ -11,21 +9,19 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import static uk.gov.companieshouse.confirmationstatementapi.ConfirmationStatementApiApplication.LOGGER;
+
 @Component
 public class LoggingInterceptor implements HandlerInterceptor{
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(LoggingInterceptor.class);
-
     @Override
     public boolean preHandle(HttpServletRequest request, HttpServletResponse response, Object handler) {
         Long startTime = System.currentTimeMillis();
         request.getSession().setAttribute("start-time", startTime);
 
         MDC.put("context", requestId(request));
         MDC.put("namespace", ConfirmationStatementApiApplication.APP_NAME);
```

Comment:
the namespace line can be removed as the imported logger has it's own namespace.